File: src/routes/resume/_education.ts
export interface Education {
  institution: string;
  degree: string;
  field: string;
  start: string;
  end: string;
  description?: string;
}

export default [
  {
    institution: "California State University",
    degree: "Computer Science",
    field: "Computer Science",
    start: "2005",
    end: "2008",
    description: "Some upper division coursework completed",
  },
] as Education[];

File: yew/src/data/education.rs
use crate::models::Education;

pub fn get_education() -> Vec<Education> {
    vec![Education {
        institution: "California State University".to_string(),
        degree: "Computer Science".to_string(),
        field: "Computer Science".to_string(),
        start_year: 2005,
        end_year: Some(2008),
        description: "Some upper division coursework completed".to_string(),
    }]
}
